src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -16,11 +16,14 @@
 
 package com.microsoft.hyperspace.index
 
-import org.apache.hadoop.fs.FileStatus
+import java.io.FileNotFoundException
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileStatus, Path}
 import org.apache.spark.sql.types.{DataType, StructType}
 
 import com.microsoft.hyperspace.actions.Constants
-import com.microsoft.hyperspace.index.Content.Directory.FileInfo
 
 // IndexLogEntry-specific fingerprint to be temporarily used where fingerprint is not defined.
 case class NoOpFingerprint() {
```

Comment:
remove this if not used? else use it in `Content` object

Reply:
good point. I would put it inside `Content` not to forget to fingerprint this in the future.

Reply:
thanks @imback82 , done